### DISCUSSION TITLE
How to iterate media files?

### DISCUSSION THREAD

**Discussion by vaniwiki** [combostrap/combo#104] [Q&A]
I want to include something like this into my iterator: `<box class="audio">{{:audio:$name.mp3|$title}}</box>` where the audio in question just resides in a folder, and is named like the document. I assume that the variables cannot be used in this way, but there may be another way to do it? Or an audio meta tag?

**Comment by gerardnico** [comment]
Unfortunately I don't sync the media in the database.

Dokuwiki on file is good but a database in front would have been better.

**Reply by vaniwiki** [reply]
Okay, this is anyways one of those things that would be really nice to have, but not really needed.

My idea was to make small audio file excerpts. The full audio will reside on the page, but I could iterate with audio excerpts matching up with descriptions. However, I have a million things to do before I get there, so we'll cross that bridge later. :)
